packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Writers/CodeWriter.cs

```diff
@@ -1176,15 +1170,19 @@ public string ToString(bool header)
             {
                 return string.Empty;
             }
-
             var builder = new StringBuilder(_length);
             IEnumerable<string> namespaces = _usingNamespaces
                 .OrderByDescending(ns => ns.StartsWith("System"))
                 .ThenBy(ns => ns, StringComparer.Ordinal);
             if (header)
             {
                 string licenseString = CodeModelPlugin.Instance.CodeWriterExtensionMethods.LicenseString;
-                builder.Append(licenseString);
+                if (!string.IsNullOrEmpty(licenseString))
+                {
+                    builder.Append(licenseString);
+                    builder.Append(_newLine);
+                    builder.Append(_newLine);
+                }
```

Comment:
I found we did not add new lines after the license string, therefore when there was a license string, we get this:
```
// license string// <auto-generated/>
```
which is not correct. We have this case in test cases.

We should only write the license string when it is not empty as well as the new lines.